### DISCUSSION TITLE
User Accounts

### DISCUSSION THREAD

**Discussion by mskenderian-wps** [fastapi-users/fastapi-users#502] [Q&A]
How can I handle User accounts? 
Using this great package with support for User Accounts.

**Comment by frankie567** [comment]
Hello!

I don't think I get what you mean. FastAPI Users provides you the logic to manage users in your application (registration, authentication...). So I think this is what you want?

**Reply by mskenderian-wps** [reply]
Yes I understand that. But any account management? Create Account, Create user and put it in account. Many users to one account. Suspend the account, all users assiociated with that account will not be able to login.

**Reply by frankie567** [reply]
What do you mean by "account"? I think there is a misunderstanding there.

**Reply by mskenderian-wps** [reply]
It looks like the answer is no.
I was asking of this package has support for users in an account.

So when a user logs in they are apart of an account. Accounts have many users. They are grouped together. Account owners can make other users for their account.

**Reply by frankie567** [reply]
Ok, I think I get it. Some may call it "Companies" or "Organizations" (like in GitHub). So, no, this is not something supported by the library.

But this should be quite straightforward to implement in your application 🙂

**Comment by mskenderian-wps** [comment]
Yes, now we are on the same page.

Ok thank you, i got it.

On Mon, Feb 8, 2021 at 9:22 AM François Voron <notifications@github.com>
wrote:

> Ok, I think I get it. Some may call it "Companies" or "Organizations"
> (like in GitHub). So, no, this is not something supported by the library.
>
> But this should be quite straightforward to implement in your application
> 🙂
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frankie567/fastapi-users/discussions/502#discussioncomment-350247>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHDL3IV2AEAGQMH4ASTVNTLS6AMW7ANCNFSM4XHU7PVA>
> .
>


-- 
-Michael Skenderian
